lib/import/hmcts_draft_creator.rb

```diff
@@ -0,0 +1,18 @@
+module Import
+  class HmctsDraftCreator
+    def create_drafts(csv_path, pause_seconds)
+      CSV.read(csv_path, headers: true).each do |row|
+        publication_id = row["publication whitehall ID"]
+        publication = Publication.find_by(id: publication_id)
+
+        puts "Creating draft preview of #{publication_id}: '#{publication.slug}'"
+
+        Whitehall.edition_services.draft_updater(publication).perform!
+
+        # Prevent the HMCTS import from overloading whitehall's publishing
+        # queue, blocking other documents from being published.
+        sleep(pause_seconds)
```

Comment:
You could provide a default for "pause_seconds", but it's optional.

Reply:
If it's a one time thing, then maybe it's not worth adding a default. But if it's going to be re-used, then it might be useful to type one less thing if the pause default to x seconds.

Reply:
Yeah, it's just for this HMCTS import. We'll remove the code afterwards. I also don't know what the default should be yet. :)